core/src/main/java/tachyon/worker/hierarchy/StorageDir.java

```diff
@@ -117,6 +128,7 @@ public void accessBlock(long blockId) {
     synchronized (mLastBlockAccessTimeMs) {
       if (containsBlock(blockId)) {
         mLastBlockAccessTimeMs.put(blockId, System.currentTimeMillis());
+        updateReferenceFrequency(blockId, REFERENCE_FREQUENCY_INCREMENT);
```

Comment:
I see. I'll add two methods. `InitReferenceFrequency(blockId, refFreq)` for adding block, and `IncrementReferenceFrequency(blockId)` for accessing block.